### PR TITLE
Version updates

### PR DESCRIPTION
**Pull request by charhate**
To fix build errors like :

```
ERROR: flake8 3.5.0 has requirement pyflakes<1.7.0,>=1.5.0, but you'll have pyflakes 2.2.0 which is incompatible.
ERROR: black 20.8b1 has requirement click>=7.1.2, but you'll have click 7.1.1 which is incompatible.
```

### CODE REVIEW

File: lemur/certificates/service.py [hunk 1]
```diff
@@ -150,7 +151,7 @@ def get_all_certs_attached_to_endpoint_without_autorotate():
         """
     return (
         Certificate.query.filter(Certificate.endpoints.any())
-        .filter(Certificate.rotation == False)
+        .filter(Certificate.rotation == false())
```
Comment (charhate) [inline lemur/certificates/service.py:154]: lint does not like `X == boolean` comparision, which is needed in filter. Thus using `sqlalchemy.sql.expression`

File: lemur/certificates/utils.py [hunk 2]
```diff
@@ -82,4 +82,4 @@ def get_key_type_from_csr(data):
             raise Exception("Unsupported key type")
 
     except NotImplemented:
-        raise NotImplemented()
+        raise NotImplementedError
```
Comment (charhate) [inline lemur/certificates/utils.py:85]: for lint error

File: requirements-dev.txt [hunk 3]
```diff
@@ -15,7 +15,7 @@ cryptography==3.2.1       # via secretstorage
 distlib==0.3.0            # via virtualenv
 docutils==0.16            # via readme-renderer
 filelock==3.0.12          # via virtualenv
-flake8==3.5.0             # via -r requirements-dev.in
+flake8==3.8.0             # via -r requirements-dev.in
```
Comment (hosseinsh) [inline requirements-dev.txt:18]: this file is auto-generated based on the input in
https://github.com/Netflix/lemur/blob/7ea6e2059a6cacf4284cf901de26ab335d6775f2/requirements-dev.in#L3

once the requirement is modified, one can either run 
```
make up-reqs
```
to compute the requirements-dev.txt

or delegate to dependbot to file a PR

**Comment by hosseinsh** [comment]
nice to see these errors go away

File: requirements-dev.txt (hunk 3 above)
Comment (charhate) [reply requirements-dev.txt:18]: Thanks for the details, flake8 update needed more version update and lint fixes. So I will do the manual steps and update the PR.

**Review by hosseinsh — APPROVED** [review]

**Pull request merged**